apps/rush-lib/src/logic/InstallManager.ts

```diff
@@ -1029,6 +1064,49 @@ export class InstallManager {
     });
   }
 
+  private _mergeEnvironmentVariables(
+    baseEnv: NodeJS.ProcessEnv,
+    environmentVariables?: IConfigurationEnvironment
+  ): NodeJS.ProcessEnv {
+    const packageManagerEnv: NodeJS.ProcessEnv = baseEnv;
+
+    if (environmentVariables) {
+      // eslint-disable-next-line guard-for-in
+      for (const envVar in environmentVariables) {
+        let setEnvironmentVariable: boolean = true;
+        console.log(`\nProcessing definition for environment variable: ${envVar}`);
+
+        if (process.env[envVar]) {
+          setEnvironmentVariable = false;
+          console.log(colors.yellow(`WARNING: Environment variable already defined:`));
+          console.log(`  Name: ${envVar}`);
+          console.log(`  Value set on the device: ${process.env[envVar]}`);
+          console.log(`  Value set in rush.json: ${environmentVariables[envVar].value}`);
+
+          if (environmentVariables[envVar].override &&
+            environmentVariables[envVar].override === true) {
```

Comment:
```suggestion
          if (environmentVariables[envVar].override) {
```

Reply:
It is an optional parameter. Is it safe to drop the null check and strict comparison?

Reply:
Yeah - it defaults to false, so if it's undefined, this will evaluate to false.